### PR TITLE
PR: Change footer to match main website

### LINKED ISSUE #56: Update footer following the changes made in the main website
## Issue Description

Change the footer to be the same as in teh main website

### PR DESCRIPTION
**Pull request by dalthviz**
### Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 3.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below
* [x] Noted what issue(s) this pull request resolves, creating one if needed

## Description of Changes

Change footer to follow main website one.

### Issue(s) Resolved

Fixes #56

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'Spyder'
-copyright = '2009–2018 The Spyder Doc Contributors'
+copyright = "<p>&copy; 2009-2018 The Spyder Doc Contributors <br> Icons by <a href='http://www.freepik.com' target='_blank' title='Freepik'>Freepik</a> from <a href='https://www.flaticon.com/' target='_blank' title='Flaticon'>Flaticon.com</a> | <a href='https://freehtml5.co/icon-free-website-template-using-bootstrap/' target='_blank'>Template</a> designed by <a href='http://freehtml5.co/' target='_blank'>FreeHTML5.co</a> <br>Hugo port by <a href='https://github.com/SteveLane/' target='_blank'>SteveLane</a> | Lektor port by <a href='https://github.com/dalthviz/' target='_blank'>Dalthviz</a></p>"
```
Comment (CAM-Gerlach) [inline doc/conf.py]: I'll do a full review in a few hours, but this should be the text that's currently present in the current footer, perhaps with "; licensed [MIT](https://opensource.org/licenses/MIT) tacked on to "Spyder Doc Contributors."

image

**Comment by ccordoba12** [comment]
@dalthviz, is this the exact same footer as the one in our main website?

**Comment by ccordoba12** [comment]
I mean, it has the same font size and its links are bold and in white?

**Comment by dalthviz** [comment]
@ccordoba12 yep, it takes the `styles.css` from the main site :+1:

**Pull request merged**